projects/common/models/src/msp-image.ts

```diff
@@ -0,0 +1,73 @@
+import {UUID} from 'angular2-uuid';
```

Comment:
Delete this file, it already exists. Take a look at `projects/common/images/src/images.ts`. I re-named all the files to "CommonImage" instead of "MSPImage"

When importing from the library these are all under `moh-common-lib/images`